pkg/virt-launcher/virtwrap/converter/converter.go

```diff
@@ -65,15 +65,11 @@ import (
 
 const deviceTypeNotCompatibleFmt = "device %s is of type lun. Not compatible with a file based disk"
 
-type HostDeviceType string
```

Comment:
This can be considered a different subject, as the centralization of the constants names could be used and still keep this one intact. I will be better IMO to introduce this in a separate commit, explaining why it can be removed.

Reply:
Created https://github.com/kubevirt/kubevirt/pull/7789